.circleci/config.yml

```diff
@@ -19,6 +19,14 @@ parameters:
     type: string
     # solbuildpackpusher/solidity-buildpack-deps:ubuntu2204.clang-8
     default: "solbuildpackpusher/solidity-buildpack-deps@sha256:2662376fe0e1ec2d346495a19a6d64508c1048d5a7325d8600c33c343fa64a0f"
+  ubuntu-2404-docker-image:
+    type: string
+    # solbuildpackpusher/solidity-buildpack-deps:ubuntu2404-1
+    default: "solbuildpackpusher/solidity-buildpack-deps@sha256:5d6d27551104321a30326d6024bd4e96d9d899a97a78eb9feea364996f1d18b5"
+  ubuntu-2404-clang-docker-image:
+    type: string
+    # solbuildpackpusher/solidity-buildpack-deps:ubuntu2404.clang-1
+    default: "solbuildpackpusher/solidity-buildpack-deps@sha256:8df0086907cc1e57068ad48841f2284a87bfbd0a95006286a19a7132d84bb861"
```

Comment:
And this image comes from here: https://github.com/ethereum/solidity/pull/15370#issuecomment-2324605800

Reply:
By the way, I think it's a good convention to hide the comments with older ones we ended up not using by marking them as "outdated".